src/Adapters/FHIRPatientAdapter.php

```diff
@@ -452,6 +465,22 @@ public function interfaceToModel( PatientInterface $patient )
         $fhirPatient->addTelecom( $email );
 
 
+        $address = new FHIRAddress();
+        $line = new FHIRString();
+        $line->setValue($patient->getStreet());
```

Comment:
I do not fully understand you.
Do you mean, I should return only first line from array?
like this:
        $street = $patient->getStreet();
        $line->setValue($street[0]);
        $address->addLine($line);